elastic/kibana
# [Monitoring] Migrate Elasticsearch ML Jobs View from Angular

## Summary

Closes #113646

Migrates the Elasticsearch/Machine Learning Jobs view to React.

This required me to create a new `ElasticsearchMLJobs` React component, as the Angular view uses its own unique directive for unclear reasons and didn't follow the standard of other views.

To test:

- Start Metricbeat
- Create at least one Machine Learning job (you can create 3 at a time using the very first Metricbeat option; try doing this twice to test pagination)
- Open the ML Jobs tab and ensure all functionality is the same when enabling/disabling `monitoring.ui.render_react_app: true`
